packages-ts/eqlabs-multisig-integration/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@chainlink/starknet-integration-tests-eqlabs-multisig",
+  "version": "1.0.0",
+  "description": "Multisig integration test",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "npx hardhat test"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.5",
+    "@chainlink-dev/starknet-multisig": "^0.1.0",
+    "@shardlabs/starknet-hardhat-plugin": "^0.6.2",
+    "@types/chai": "^4.3.1",
+    "@types/elliptic": "^6.4.14",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.11.10",
+    "chai": "^4.3.4",
+    "ethers": "^5.6.8",
+    "ts-node": "^10.7.0",
+    "typescript": "^4.7.2"
+  },
+  "dependencies": {
+    "@toruslabs/starkware-crypto": "^1.0.0",
```

Comment:
Do we use this dependency for anything here?

Reply:
No, removed